tests/unit/test_sandbox.py

```diff
@@ -36,9 +35,23 @@ def temp_dir(monkeypatch):
 
 def test_env_vars(temp_dir):
     os.environ['SANDBOX_ENV_FOOBAR'] = 'BAZ'
+    ssh_box_config = AppConfig(
```

Comment:
why not move this into the `create_docker_box_from_app_config` method or even outside/above first test?